CMakeLists.txt

```diff
@@ -224,10 +247,16 @@ foreach(TEST_SOURCE ${TEST_SOURCE_LIST})
            "--suppressions=${CURRENT_GLOBAL_SUPPRESSION}")
     endforeach()
     list(APPEND CURRENT_MEMTEST_COMMAND "${PREFIX_PATH}/${TEST_COMMAND}")
-    add_test(NAME memtest_${TEST_TARGET} COMMAND ${CURRENT_MEMTEST_COMMAND})
-    set_property(TEST memtest_${TEST_TARGET} PROPERTY LABELS memtest)
-    set_tests_properties(
-      memtest_${TEST_TARGET}
-      PROPERTIES ENVIRONMENT "SKETCHER_SOURCE_DIR=${CMAKE_SOURCE_DIR}")
+    set(TEST_TARGET "memtest_${TEST_TARGET}")
+    set(TEST_COMMAND "${CURRENT_MEMTEST_COMMAND}")
+  endif()
+  # Add the test to CTest
+  add_test(NAME ${TEST_TARGET} COMMAND ${TEST_COMMAND})
```

Comment:
Is the goal of the changes here to have the memtest build run the Valgrind versions of the tests _instead of_ the the non-Valgrind tests, as opposed to _in addition to_?  (I have no objection to that; I just wanted to make sure that I was reading this correctly.)